### PR TITLE
[zh] sync /kubeadm/kubeadm-reconfigure.md

### LINKED ISSUE #37752: Synchronize the latest English documents
Path: content/zh cn/docs/tasks/administrator cluster/kubeadm/kubeadm reconfigure.md

### PR DESCRIPTION
**Pull request by xzyangPlus**
Fix #37752

### CODE REVIEW

File: content/zh-cn/docs/tasks/administer-cluster/kubeadm/kubeadm-reconfigure.md [hunk 1]
```diff
@@ -496,14 +496,16 @@ kubelet 标志会覆盖相关的 `KubeletConfiguration` 选项，但请注意，
...
 - [Certificate management with kubeadm](/docs/tasks/administer-cluster/kubeadm/kubeadm-certs)
+- [Find more about kubeadm set-up](/docs/reference/setup-tools/kubeadm/)
 -->
 
 - [升级 kubeadm 集群](/zh-cn/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade)
 - [使用 kubeadm API 自定义组件](/zh-cn/docs/setup/production-environment/tools/kubeadm/control-plane-flags)
-- [使用 kubeadm 管理证书](/zh-cn/docs/tasks/administer-cluster/kubeadm/kubeadm-certs)
\ No newline at end of file
+- [使用 kubeadm 管理证书](/zh-cn/docs/tasks/administer-cluster/kubeadm/kubeadm-certs)
+- [了解有关kubeadm设置的更多信息](/docs/reference/setup-tools/kubeadm/)
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/administer-cluster/kubeadm/kubeadm-reconfigure.md:511]:
```suggestion
- [进一步了解 kubeadm 设置](/zh-cn/docs/reference/setup-tools/kubeadm/)  
```

File: content/zh-cn/docs/tasks/administer-cluster/kubeadm/kubeadm-reconfigure.md [hunk 2]
```diff
@@ -496,14 +496,16 @@ kubelet 标志会覆盖相关的 `KubeletConfiguration` 选项，但请注意，
 更改 `/var/lib/kubelet/config.conf` 或 `/var/lib/kubelet/kubeadm-flags.env` 
 后需要重启 kubelet。
 
-{{% heading "whatsnext" %}}
 
+{{% heading "whatsnext" %}}
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/administer-cluster/kubeadm/kubeadm-reconfigure.md:500]:
```suggestion
## {{% heading "whatsnext" %}}
```

**Comment by windsonsea** [comment]
Could you please squash the commits into one?

**Comment by xzyangPlus** [comment]
> Could you please squash the commits into one?
what should I do

**Comment by windsonsea** [comment]
Contribute to the k8s/website and refer to this site:
https://kubernetes.io/docs/contribute/new-content/open-a-pr/#squashing-commits

**Comment by xzyangPlus** [comment]
> Could you please squash the commits into one?

done

**Pull request merged**